app/src/main/java/com/duckduckgo/app/global/db/AppDatabase.kt

```diff
@@ -20,16 +20,20 @@ import android.arch.persistence.room.Database
 import android.arch.persistence.room.RoomDatabase
 import com.duckduckgo.app.httpsupgrade.db.HttpsUpgradeDomain
 import com.duckduckgo.app.httpsupgrade.db.HttpsUpgradeDomainDao
+import com.duckduckgo.app.settings.db.AppConfigurationDao
+import com.duckduckgo.app.settings.db.AppConfigurationEntity
 import com.duckduckgo.app.trackerdetection.db.TrackerDataDao
 import com.duckduckgo.app.trackerdetection.model.DisconnectTracker
 
 @Database(exportSchema = true, version = 2, entities = [
```

Comment:
I'm getting a crash on upgrades. Can we bump the version number?

Reply:
Good point!